CHANGELOG.md

```diff
@@ -25,6 +25,8 @@
   - snowflake.snowpark.functions
     - current_database
     - current_session
+    - concat
+    - concat_ws
```

Comment:
nit: you probably need to rebase the main branch and move the changelog to 1.16?